CheckDrive.Api/CheckDrive.Infrastructure/Persistence/CheckDriveDbContext.cs

```diff
@@ -28,10 +30,10 @@ public class CheckDriveDbContext : IdentityDbContext, ICheckDriveDbContext
     public CheckDriveDbContext(DbContextOptions<CheckDriveDbContext> options)
         : base(options)
     {
-        Database.EnsureCreated();
-        // Database.Migrate();
+        //Database.EnsureCreated();
+        //Database.Migrate();
 
-        SaveChangesAsync();
+        //SaveChangesAsync();
```

Comment:
Production code should not have any commented code unless it is necessary

Reply:
Can I  remove this codes from file ?